hasher-matcher-actioner/hmalib/lambdas/actions/writebacker.py

```diff
@@ -44,4 +45,14 @@ def lambda_handler(event, context):
 
 
 if __name__ == "__main__":
-    pass
+    # This will react to real descriptors
```

Comment:
`MOCK_TE_API` is a different flag for unit tests. 

This pattern is pretty similar to those in other `__name__ == "__main__"` blocks such as in pdq_indexer. For basic debugging. I can document to make it clearer and if you feel strongly can add another environ flag

Reply:
Right, that flag could be set here (or even treated as a default) because we are reacting to real descriptors better documenting and preferably another level of confirmation before doing so would be ideal imo. (e.g. using https://docs.python.org/3/library/functions.html#input)